index.html

```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
         </a>
       </div>
     </header>
+    <section class="works">
+      <div class="projects">
+        <img src="images/snapshoot-project-1.png" alt="snapshoot-project-1" />
+        <p class="project-title">Tonic</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
```

Comment:
- [ ] Please know that the project description should be wrapped in a `p` tag, kindly change that. 🙂

Reply:
**Changed all `div`s having the `.project-desc` class to `p` tag the**